modules/javafx.graphics/src/test/java/test/javafx/scene/CssStyleHelperTest.java

```diff
@@ -608,4 +609,53 @@ public void movingBranchToDifferentBranchGetsNewCssVariableTest() throws IOExcep
         assertEquals(Color.BLUE, E.backgroundProperty().getValue().getFills().get(0).getFill());
         assertEquals(Color.BLUE, F.backgroundProperty().getValue().getFills().get(0).getFill());
     }
+
+    @Test
+    public void initialNodeWithUserSetValueShouldNotResetValuesOnOtherNodesWithoutOverriddenValue() throws IOException {
+        Stylesheet stylesheet = new CssParser().parse(
+            "initialNodeWithUserSetValueShouldNotResetValuesOnOtherNodesWithoutOverridenValue",
+            """
+                .pane {
+                    -fx-padding: 4;
+                }
+            """
+        );
```

Comment:
Even if there is, this still shouln't stop us using new language features, should it? Otherwise we can't really use any new feature, as a fix *might* be backported.
On the other hand it is also easy to replace this text block when backported.

Reply:
it could also be a one liner

Reply:
This is a similar case to using pattern matching `instanceof`, which we've started using in new code. My thinking is that in new code it's should be OK to use new language features in new code, unless we know that it is a bug fix that needs to be backported.